app/src/release/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,20 @@
+<resources>
+    <!--
+    TODO: Before you release your application, you need a Google Maps API key.
+
+    To do this, you can either add your release key credentials to your existing
+    key, or create a new key.
+
+    Note that this file specifies the API key for the release build target.
+    If you have previously set up a key for the debug target with the debug signing certificate,
+    you will also need to set up a key for your release certificate.
+
+    Follow the directions here:
+
+    https://developers.google.com/maps/documentation/android/signup
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">YOUR_KEY_HERE</string>
```

Comment:
Is this the same key that goes in `app/src/debug/res/values/google_maps_api.xml` ?

Reply:
Also, after we merge this pull request, I think it'd make sense to add both google_maps_api.xml files to gitignore so we can all put our API keys locally.

Reply:
Unless I'm completely misinterpreting this, and we should just all use the same key...

Reply:
As discussed, this is a per app key, so no need to add to gitignore.